h2o-py/h2o/explanation/_explain.py

```diff
@@ -1380,10 +1428,12 @@ def ice_plot(
                          label="Partial Dependence")
 
         _add_histogram(frame, column)
-        plt.title("Individual Conditional Expectation for \"{}\"\non column \"{}\"{}".format(
+        group_label = kwargs.get('group_label')
```

Comment:
ultranit: You can provide default value, and can avoid ternary later.
```suggestion
        group_label = kwargs.get("group_label", "")
```